src/Operation/DropCollection.php

```diff
@@ -83,36 +82,28 @@ public function __construct(private string $databaseName, private string $collec
     /**
      * Execute the operation.
      *
-     * @see Executable::execute()
-     * @return array|object Command result document
      * @throws UnsupportedException if write concern is used and unsupported
      * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
      */
-    public function execute(Server $server): array|object
+    public function execute(Server $server): void
     {
         $inTransaction = isset($this->options['session']) && $this->options['session']->isInTransaction();
         if ($inTransaction && isset($this->options['writeConcern'])) {
             throw UnsupportedException::writeConcernNotSupportedInTransaction();
         }
 
         try {
-            $cursor = $server->executeWriteCommand($this->databaseName, $this->createCommand(), $this->createOptions());
+            $server->executeWriteCommand($this->databaseName, $this->createCommand(), $this->createOptions());
         } catch (CommandException $e) {
             /* The server may return an error if the collection does not exist.
              * Check for an error code and return the command reply instead of
              * throwing. */
             if ($e->getCode() === self::ERROR_CODE_NAMESPACE_NOT_FOUND) {
-                return $e->getResultDocument();
+                return;
```

Comment:
"return the command reply instead of throwing" in the comment above should probably be changed to "return instead of throwing" or "suppress the error".